lib/appsignal/transaction.rb

```diff
@@ -99,6 +99,8 @@ def set_metadata(key, value)
 
     def set_error(error)
       return unless error
+      return if Appsignal.is_ignored_error?(error)
```

Comment:
This might introduce unexpected behavior, you're now able to send an error regardless if it's ignored or not and we're changing it now.